Gemfile

```diff
@@ -36,14 +36,21 @@ gem 'net-ldap', '~> 0.16.0'
 # secure_headers provides x-frame, csp and other http headers
 gem 'secure_headers', '~> 1.4.0'
 
-gem 'faraday', '~> 0.9.0'
-gem 'faraday_middleware', '~> 0.9.1'
```

Comment:
Some ETS code used the 'Faraday' HTTP client library. CalCentral primarily relies on HTTParty.